xcm/src/v0/multi_asset.rs

```diff
@@ -61,35 +61,34 @@ pub enum AssetInstance {
 /// ### Abstract identifiers
 ///
 /// Abstract identifiers are absolute identifiers that represent a notional asset which can exist within multiple
-/// consensus systems. These tend to be simpler to deal with since their broad meaning is unchanged regardless stay
-/// of the consensus system in which it is interpreted.
+/// consensus systems. These tend to be simpler to deal with since their broad meaning is unchanged regardless stay of
```

Comment:
I tried to prevent myself from doing so many of these and reverted most 🙈 but FYI this is only properly rewrapping the doc to 120 chars.